samples/security/protection_psoc6/README.rst

```diff
@@ -0,0 +1,112 @@
+.. _prot-psoc6-sample:
+
+Sample secure domain application
+################################
+
+Overview
+********
+
+The :ref:`cy8ckit_062_wifi_bt` board has two core processors (Cortex-M4
+and Cortex-M0+). This sample application creates a demo with a protected
+area created on the Cortex-M0+ core (secure domain). On start CM0+ core
+protects its Flash, SRAM (using SMPU structures) from CM4 access and
+restricts access to some peripheral resources (using PPU structures).
+When CM4 tries to access protected SRAM area the system breaks it into
+a Hardware Fault exception.
+
+Requirements
+************
+
+- :ref:`cy8ckit_062_wifi_bt` board
+
+Building and Running
+********************
+
+#. Build the project
+
+.. code-block:: console
+
+   $ cd samples/security/protection_psoc6
+   $ mkdir build
+   $ cd build
+   $ cmake -G"Eclipse CDT4 - Ninja" ..
+   $ ninja
+
+#. Switch the DevKit into CMSIS-DAP mode using SW3 (LED2 should blink) and
+flash the board:
```

Comment:
the continuation of the numbered list item must be indented to the text in the previous line:
```suggestion
   flash the board:
```